patch/patcher.go

```diff
@@ -75,6 +75,13 @@ func (p *Patcher) scan() error {
 func (p *Patcher) scanFile(f *protogen.File) {
 	log.Printf("\nScan proto:\t%s", f.Desc.Path())
 
+	// Locally generate Go from the source proto file.
```

Comment:
This is the critical part of this PR. It uses the `internal_gengo` package to run `protoc-gen-go` in-process.